wcomponents-theme/src/test/intern/wc.file.util.test.js

```diff
@@ -0,0 +1,98 @@
+define(["intern!object", "intern/chai!assert", "./resources/test.utils!"],
+	function (registerSuite, assert, testutils) {
+		"use strict";
+		var fileUtil,
+			wcconfig;
+		registerSuite({
+			name: "wc/file/util",
+			setup: function () {
+				return testutils.setupHelper(["wc/file/util", "wc/config"], function (util, config) {
+					fileUtil = util;
+					wcconfig = config;
+				});
+			},
+			testBlobToFile: function () {
+				var expectedMime = "text/plain",
+					expectedFile = "test.file",
+					content = "This is my blob content",
+					blob = new Blob([content], {type: expectedMime}),
+					file = fileUtil.blobToFile(blob, {
+						name: expectedFile
+					});
+				assert.strictEqual(file.name, expectedFile);
+				assert.strictEqual(file.type, expectedMime);
+				assert.strictEqual(file.size, content.length);
+				// https://developer.mozilla.org/en-US/docs/Web/API/File#Properties
+				assert.isDefined(file.lastModified, "Check this is a file (not a blob).");
+				assert.strictEqual(file.lastModified, file.lastModifiedDate.getTime());
+				return new Promise(function (win) {
+					var fileReader = new FileReader();
+					fileReader.onload = function () {
+						assert.strictEqual(fileReader.result, content);
+						assert.notStrictEqual(fileReader.result, "content");
+						win();
```

Comment:
I would tend to put this in a `finally` block but it's probably ok without it - it's hard to see how the simple assertions above could fail. If they DID fail then the test runner would hang as `win` would never be called. Like I say, just food for thought, probably not necessary here.

Reply:
you mean `onload` function, to look like this?
```javascript
try {
	assert.strictEqual(fileReader.result, content);
	assert.notStrictEqual(fileReader.result, "content");
} finally {
	win();
}
```

Reply:
yep